api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java

```diff
@@ -156,8 +156,8 @@ public String getDisplayString() {
 	 * Also @see org.openmrs.Attributable#getPossibleValues()
 	 * 
 	 * @return {@link LabTest} object(s)
-	 * @deprecated Data provided by this method can be better achieved from 
-	 * appropriate service at point of use.
+	 * @deprecated Data provided by this method can be better achieved from appropriate service at point

```

Comment:
Unnecessary change, breaking the formatting

Reply:
this is related to code format, and is a result of maven clean install